### PR TITLE
Update /risc-v with new boards and 22.10 update

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Do not merge** until `Oct 25, 9am UK time`

- Update /risc-v with new boards and 22.10 update as per [copydoc](https://docs.google.com/document/d/1kkJeq9HdH3iWV35HwihcJd3CxZw9Jy21W66lJl8VbWs/edit#)

## QA

**NOTE** tab-5 is currently being omitted from the page, there is a conversation in the copydoc about this
- Go to https://ubuntu-com-12182.demos.haus/download/risc-v and compare against [copydoc](https://docs.google.com/document/d/1kkJeq9HdH3iWV35HwihcJd3CxZw9Jy21W66lJl8VbWs/edit#)

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5957
Fixes https://github.com/canonical/web-squad/issues/6090

### CODE REVIEW

File: templates/download/risc-v/tab-4.html [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
 				loading="auto"
 				) | safe
 			}}
 		</div>
 	</div>
 	<div class="u-fixed-width">
-		<h4>First time installing Ubuntu on QEMU/Unmatched?</h4>
-		<p>Please see the <a href="https://discourse.ubuntu.com/t/ubuntu-installation-on-the-sifive-hifive-unmatched-board-using-a-server-install-image/27804">Ubuntu installation on the SiFive HiFive Unmatched board using a server install image - RISC-V-docs</a> and <a href="https://discourse.ubuntu.com/t/ubuntu-installation-on-a-risc-v-virtual-machine-using-a-server-install-image-and-qemu/27636">Ubuntu installation on a RISC-V virtual machine using a server install image and QEMU</a> for more information.</p>
+		<h4>First time installing Ubuntu on the LicheeRV Dock?</h4>
+		<p>Please see the <a href="#">How to install Ubuntu on the LicheeRV Dock guide</a> for more information.</p>
```
Comment (minkyngkm) [inline templates/download/risc-v/tab-4.html:34]:
```suggestion
		<p>Please see the <a href="#">How to install Ubuntu on the LicheeRV Dock</a> for more information.</p>
```

File: templates/download/risc-v/tab-2.html [hunk 2]
```diff
@@ -2,6 +2,16 @@
 	<div class="row u-vertically-center u-no-padding">
 		<div class="col-6">
 			<h2>Download Ubuntu Server</h2>
+			<h3 class="p-heading--4">Ubuntu Server 22.10</h3>
+			<div class="p-notification--information">
+				<div class="p-notification__content">
+					<h5 class="p-notification__title">Note:</h5>
```
Comment (minkyngkm) [inline templates/download/risc-v/tab-2.html:8]: This note looks different from the same notes in the other tabs.
image
image

File: templates/download/risc-v/tab-1.html [hunk 3]
```diff
@@ -2,6 +2,16 @@
 	<div class="row u-vertically-center u-no-padding">
 		<div class="col-6">
 			<h2>Download Ubuntu Server Preinstalled Image</h2>
+			<h3 class="p-heading--4">Ubuntu Server 22.10</h3>
+			<div class="p-notification--information is-inline">
+				<div class="p-notification__content">
+					<h4 class="p-notification__title">Note:</h4>
+					<p class="p-notification__message">It is an early RISC-V developer access through Ubuntu 22.10 and this is a preinstalled image, if you have a NVMe drive, we advise you to use the Ubuntu Server live installer which can be found <a href="https://cdimage.ubuntu.com/releases/22.04.1/release/ubuntu-22.04.1-preinstalled-server-riscv64+unmatched.img.xz">here</a>.</p>
+				</div>
```
Comment (carkod) [inline templates/download/risc-v/tab-1.html:10]: This link has already been provided

Comment (petesfrench) [reply templates/download/risc-v/tab-1.html:10]: That is not the link we are waiting for. It is the link under 'How to install Ubuntu on the LicheeRV Dock'

**Pull request merged**